Microsoft.Toolkit.Win32/Microsoft.Toolkit.Sample.Wpf.XamlHost/WrappedButton.cs

```diff
@@ -0,0 +1,104 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.ComponentModel;
+using System.Windows.Media;
+using Microsoft.Toolkit.Win32.UI.XamlHost;
+using Microsoft.Toolkit.Wpf.UI.XamlHost;
+
+namespace Microsoft.Toolkit.Sample.Wpf.XamlHost
+{
+    public class WrappedButton : WindowsXamlHostBase
```

Comment:
>WindowsXamlHostBase [](start = 33, length = 19)

Why not use `HostBaseExt` that encapsulates all the logic in `OnInitialized`?

Reply:
That sounds like some kind of compiler and runtime test, not a sample...

Reply:
I would lean toward pushing the Ext functionality into the Base classes (and get rid of the Ext classes) than change these.